docs/debuginfod.md

```diff
@@ -0,0 +1,45 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries that are being profiled. Debug information
+typically comprises ELF object files, DWARF debug data, and source code
+required for symbolization. However, sometimes, application packages distributed
+by various Linux distros strip away debug information to minimize the size of
```

Comment:
I think unless you're already aware of the "strip" terminology, this can be confusing. How about we link to [strip](https://man7.org/linux/man-pages/man1/strip.1.html)?